traits/etsconfig/tests/test_etsconfig.py

```diff
@@ -258,7 +256,7 @@ def test_provisional_toolkit(self):
 
         with mock_sys_argv(test_args):
             with mock_os_environ(test_environ):
-                print(repr(self.ETSConfig.toolkit))
+                repr(self.ETSConfig.toolkit)
```

Comment:
why remove the print but keep the call to `repr`?

Reply:
To keep the effect of the test. It's (indirectly) testing that the `repr` doesn't fail, so I wanted to keep that test.